plugin/extract-aliases/index.test.js

```diff
@@ -45,13 +46,17 @@ describe('extract-aliases', () => {
   }
 
   const result = {
-    '@file': path.join(appPath, './src/file.js'),
-    '@file2': path.join(appPath, './src/file2.js'),
-    '@dir': path.join(appPath, './src/dir'),
-    '@dir2': path.join(appPath, './src/dir2'),
-    '$dir3': path.join(appPath, './src/dir3'),
-    'my-package': path.join(appPath, './node_modules/some-package'),
-    '@material-ui': path.join(
```

Comment:
By the way, I think it's better to use snapshot testing here

Reply:
> By the way, I think it's better to use snapshot testing here

Good idea, I'll try it soon